css/project.css

```diff
@@ -0,0 +1,65 @@
+.navigation {
+    background-color: var(--text-color) !important;
+    color: var(--secondary-color) !important;
+}
+
+.navigation .link {
+    color: var(--secondary-color) !important;
```

Comment:
Kindly avoid styles with important. You can try using a higher specificity.

Reply:
Will look into this. Thank you for suggestions.

Reply:
Updated with specificity. It is very easy to use. Thanks for reminding